types/v2/structure.go

```diff
@@ -27,12 +27,14 @@ import (
 type StructureTest struct {
 	DriverImpl         func(drivers.DriverConfig) (drivers.Driver, error)
 	DriverArgs         drivers.DriverConfig
-	GlobalEnvVars      []unversioned.EnvVar `yaml:"globalEnvVars"`
-	CommandTests       []CommandTest        `yaml:"commandTests"`
-	FileExistenceTests []FileExistenceTest  `yaml:"fileExistenceTests"`
-	FileContentTests   []FileContentTest    `yaml:"fileContentTests"`
-	MetadataTest       MetadataTest         `yaml:"metadataTest"`
-	LicenseTests       []LicenseTest        `yaml:"licenseTests"`
+	Driver             string                   `yaml:"driver"`
```

Comment:
Again, as above. While i've added this. I'm not sure what to do with it now that it's there. Let me know, and i'll make the required changes :)

Reply:
You actually probably don't want to put this here. Check out the example I linked above for the schemaVersion, it should look very similar to that 👍